src/js/edu-benefits/1990-rjsf/BenefitsRelinquishmentField.jsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash/fp';
+
+/**
+ * Field for selecting the benefitsRelinquished and corresponding date as necessary.
+ * @type {Object}
+ */
+export default class BenefitsRelinquishmentField extends React.Component {
+  onPropertyChange(name) {
+    return (value) => {
+      const formData = Object.keys(this.props.formData || {}).length
+        ? this.props.formData
+        : {
```

Comment:
I don't know that this is _really_ necessary, but it's just some code I took from `ObjectField` and modified.